src/testsuite/vec_perf_i128.c

```diff
@@ -35,10 +35,17 @@
 #include <testsuite/vec_perf_i128.h>
 
 extern const vui128_t vtipowof10 [];
+static const vui32_t c_zero = CONST_VINT32_W(0, 0, 0, 0);
 static const vui32_t c_one = CONST_VINT32_W(0, 0, 0, 1);
 static const vui32_t c_two = CONST_VINT32_W(0, 0, 0, 2);
+static const vui32_t c_nine = CONST_VINT32_W(0, 0, 0, 9);
 static const vui32_t c_ten = CONST_VINT32_W(0, 0, 0, 10);
 
+static const vui32_t ten_64h =
+    CONST_VINT32_W(0x00000000, 0x00184f03, 0xe93ff9f4, 0xdaa797ed);
+static const vui32_t ten_64l =
+    CONST_VINT32_W(0x6e38ed64, 0xbf6a1f01, 0x00000000, 0x00000000);
+
```

Comment:
it could help to explain what these are.  is there a better, more readable/maintainable way to express these?